algorithmic_efficiency/workloads/wmt/wmt_jax/models.py

```diff
@@ -192,15 +204,16 @@ def __call__(self, inputs, encoder_mask=None):
       output after transformer encoder block.
     """
     cfg = self.config
+    norm_first = cfg.norm_first
```

Comment:
Nit: can we change 'norm_first' to 'pre_ln', I think 'pre_ln' and 'post_ln' are the terminology we've been using.

Reply:
Done.